tests/integration_tests/test_sim.py

```diff
@@ -134,6 +136,18 @@ def test_pin_mode(self, qtbot, sim_app, device_type):
         assert hw.get_pin_mode(17) == PinModes.INPUT
         assert hw.get_pin_mode(34) == PinModes.OUTPUT
 
+    def test_voltage_reading(self, qtbot, sim_app, device_type):
+        flush_packets(sim_app, device_type)
+
+        snapshot = get_event_stats_snapshot()
+        sim_app.send_command(device_type.name + ".VOLT")
+        assert SINGLE_SENSOR_EVENT.wait(snapshot) == 1
+
+        last_voltage = sim_app.rocket_data.last_value_by_device(device_type, DataEntryIds.VOLTAGE)
+        assert(round(last_voltage, 1) == VoltageSensor.NOMINAL_VOLTAGE)
```

Comment:
I think it would be worth after these lines, changing the voltage in VoltageSensor and then repeating these lines again checking for the new value. 

This way we ensure that the firmware is updating the value and isn't just sending the same thing every time.